lib/node_modules/@stdlib/array/complex64/docs/types/index.d.ts

```diff
@@ -813,6 +857,39 @@ declare class Complex64Array implements Complex64ArrayInterface {
 	*/
 	map<U = unknown>( fcn: MapFcn<U>, thisArg?: ThisParameterType<MapFcn<U>> ): Complex64Array;
 
+	/**
+	* Returns value that results from running the reducer callback function to completion over the entire typed array.
+	*
+	* @param reducer - callback function
+	* @param initialValue - initial value
+	* @returns accumulated result
+	*
+	* @example
+	* var realf = require( '@stdlib/complex/realf' );
+	* var imagf = require( '@stdlib/complex/imagf' );
+	* var Complex64 = require( '@stdlib/complex/float32' );
+	*
+	* function sum( acc, v ) {
+	*     return new Complex64( realf( acc ) + realf( v ), imagf( acc ) + imagf( v ) );
+	* }
```

Comment:
You can use `@stdlib/math/base/ops/caddf`, I believe, rather than inline a function.